cloudant-sync-datastore-android-encryption/src/main/java/com/cloudant/sync/sqlite/android/AndroidSQLCipherSQLite.java

```diff
@@ -47,7 +49,7 @@
      * @param provider Provider object that contains the key to encrypt the SQLCipher database
      * @return
      */
-    public static AndroidSQLCipherSQLite createAndroidSQLite(String path, KeyProvider provider) {
+    public static AndroidSQLCipherSQLite openOrCreate(File path, KeyProvider provider) {
```

Comment:
The name is an attempt to be consistent with the call to the underlying SQLite java interface - `SQLiteDatabase.openOrCreateDatabase`, and `SQLiteConnection.open(true)` where the boolean flag is called `allowCreate`. I'm happy to use a different name, but it should be the same across our 3 classes!